script/configs/hem_config.yaml

```diff
@@ -0,0 +1,89 @@
+# ------------------------------------------------------------------------------
```

Comment:
Moved some options into their own subsections. This makes the interface cleaner.